pkgs/applications/video/clapper/default.nix

```diff
@@ -58,20 +54,19 @@ stdenv.mkDerivation rec {
     gtk4
     libGL
     libadwaita
-    libsoup
+    libsoup_3
     wayland
     wayland-protocols
+    vala
+    libmicrodns
+    desktop-file-utils # for update-desktop-database
+    shared-mime-info # for update-mime-database
```

Comment:
These two should go to `nativeBuildInputs` as they are executed at build time.

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/306548/commits/2fd6717b51137e2ab74b11cb4117332e3f69e4ad